internal/eval/eval.go

```diff
@@ -36,12 +36,16 @@ const (
 	Initialized
 )
 
-var TxnCommands map[string]bool
-var serverID string
-var diceCommandsCount int
+var (
```

Comment:
@JyotinderSingh I use neovim and it uses gofmt to autoformat these. Is this okay or should I disable my formatter?

Reply:
It's okay, these changes look good for now.